app/Export/GetImages.hs

```diff
@@ -26,7 +27,7 @@ getActiveGraphImage req = do
         graphName =
             replace "-" " " $
                 maybe "Computer-Science" cookieValue (M.lookup "active-graph" cookies)
-    getGraphImage graphName (M.map cookieValue cookies)
+    getGraphImage graphName (M.mapKeys T.pack $ M.map cookieValue cookies)
```

Comment:
No need to use `T.pack` here - use `readCookieValue` instead: http://hackage.haskell.org/package/happstack-server-7.4.6.2/docs/Happstack-Server-RqData.html#v:lookCookieValue